src/main/java/apoc/convert/UnsupportedTypeException.java

```diff
@@ -0,0 +1,9 @@
+package apoc.convert;
+
+public class UnsupportedTypeException extends RuntimeException {
```

Comment:
don't need that, runtime exception is good enough. the end user will get just string message anyway